docusaurus/docs/iOS/guides/push-notifications.md

```diff
@@ -1,3 +1,179 @@
+     * Click **Continue** and save the file on your hard drive in secure area
+
+## Step 2. Creating a Push Notification SSL Certificate
+
+1. Sign in to your [Apple Developer Account](https://developer.apple.com/account/) and then navigate to **Certificates, Identifiers & Providers**
+2. Go to **Certificates** tab
+3. Click on the **+** button to Add a new certificate
+4. In the **Services** section, select Apple Push Notification service SSL (Sandbox) and then click on Continue
+5. Select your app in the dropdown list and then click on **Continue**
+6. You will see instructions on how to generate a `.certSigningRequest` file. This was already covered in the previous section. Click on **Continue**
+7. Click on **Choose File** and then navigate to where you have saved the `.certSigningRequest` file from the previous section, then click on **Continue**
+8. Click on **Download** to save your certificate to your hard drive
+
+## Step 3. Export the Certificate in .p12 Format
+
+1. On your mac, navigate to where you have saved the `.cer` file from the previous section and double click on the file. This will add it to your macOS Keychain.
+2. Go to **Keychain Access**
+3. At the top left, select **Keychains > Login**
+4. Then, at the bottom left, select **Category > Certificates**
+5. Select the certificate you've created in the previous step. It should look like `Apple Development IOS Push Services: YOUR_APP_NAME` and expand it to see the private key(it should be named after the Name you provided when creating the `Certificate Signing Request`
+6. Right-click the private key and click on **Export**. In the **File** format section select **Personal Information Exchange (.p12)** and save the file on your hard drive
+  
+## Step 4. Uploading the Certificate to Stream Chat
+
+Certificate uploading process is the same regardless of authentication method and was [covered in detail above](#step-4-uploading-the-key-credentials-to-stream-chat).
+
+Essentially you just need to run a setup wizzard:
+
+    stream chat:push:apn
+
+
+## Testing Push Notifications setup
+
+First of all make sure you have at least one [device associated](#managing-users-for-testing-purposes).
+
+If the device you want to test notifications on is already added to the list, you can now test pushes:
+
+    stream chat:push:test
+
+This will do several things for you:
+
+1. Pick a random message from a channel that this user is part of
+2. Use the notification templates configured for your push providers to render the payload using this message
+3. Send this payload to all of the user's devices
+   
+## Managing users for testing purposes
+
+For testing purposes you can manage users using Stream CLI:
+
+### Adding users
+    stream chat:push:device:add
+
+
+### Removing users
+    stream chat:push:device:delete
+
+### Getting the list of all devices registered for pushes
+    stream chat:push:device:get
+
+## Managing users in client code
+
+### Adding users
```

Comment:
This is explaining devices and not users. Also, needs explanation "Why/when do we need to add/remove device?"

Reply:
Thank you, fixed 👌